tools/topology/CMakeLists.txt

```diff
@@ -69,7 +69,8 @@ set(TPLGS
 	"sof-apl-wm8804\;sof-apl-wm8804"
 	"sof-apl-da7219\;sof-apl-da7219"
 	"sof-glk-da7219-kwd\;sof-glk-da7219-kwd"
-	"sof-glk-da7219\;sof-glk-da7219"
+	"sof-glk-da7219\;sof-glk-da7219\;-DCPROC=volume"
+	"sof-glk-da7219\;sof-glk-eq-da7219\;-DCPROC=eq"
```

Comment:
Could we have different macro names for endpoints? Like DMICPROC, SPKPROC, HSMICPROC, HSEARPROC. If not defined it could default to volume. If volume need to be removed from pipeline it could be set to passthrough This way we could set from top level e.g. the speaker pipeline to pipe-eq-iir-volume-playback.m4. 

So in this case the set macro would be DMICPROC to know at makefile level what is impacted.

Reply:
A good point here.

Reply:
I wrote more about this into your other PR.

Reply:
Will GLK (GLK_BOB_DA7219) test setup be updated with this topology as the main one once merged?

Reply:
@mmaka1 I don't so. There's need to have 0 dB gain in DMIC capture path and this topology sets in addition to 50 Hz high-pass a +20 dB gain.

When OK from @cujomalainey and @sebcarlucci I think we should use as default DMICPROC dcblock-volume.

Reply:
Agreed, as currently stands, we don't have a use for the EQ from an input standpoint. We still have to verify the differences between our floating point and fixed point algorithms. We are hoping that will be soon. Once we understand that then we can look at moving the dcblocker into the dmic path.

Reply:
@singalsu @cujomalainey Yes and it was my question, what kind of GLK+EQ testing the PR description was talking about and whether any actions in CI setups were planned.

Reply:
This is part of efforts to expand test coverage. DMIC testing was done in APL UP2 with DMIC kit which need some steps to install it, some debugging is required when wire connection is loosened. GLK bobba is better and stable than APL UP+DMIC kit.